src/pages/guides/using/aem/aem_healthcheck_servlet.md

```diff
@@ -0,0 +1,47 @@
+# Trigger Adobe I/O Events health checks
+
```

Comment:
Shall we add a line saying that this is applicable only for `AEM as a Cloud Service` & `AEM 6.5 and later version`?

Reply:
yes